nextjs-oneclick/temporal/lib/activities.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+exports.canceledPurchase = exports.checkoutItem = void 0;
+// @@@SNIPSTART nodejs-hello-activity
```

Comment:
Get rid of this snip before merging please. Also, let's not commit compiled TS code.